relayer-cli/src/utils/relay.ts

```diff
@@ -35,8 +35,14 @@ const relay = async (chainId: number, nonce: number) => {
     getMessageDataToRelay(chainId, nonce),
   ]);
 
-  const txn = await veaOutbox.sendMessage(proof, nonce, to, data);
-  await txn.wait();
+  try {
+    const txn = await veaOutbox.sendMessage(proof, nonce, to, data);
+    const receipt = await txn.wait();
+    return receipt;
+  } catch (error) {
```

Comment:
the try/catch can be removed so the code is simpler and there is one branch less to test, just ensure that the function is small and the name is easy to find so we can quickly understand what is failing from the stack trace